state-chain/chains/src/lib.rs

```diff
@@ -488,3 +488,21 @@ impl<C: Chain> FeeEstimationApi<C> for () {
 		Default::default()
 	}
 }
+
+/// Defines an interface for a retry policy.
+pub trait RetryPolicy {
+	type BlockNumber;
+	type AttemptCount;
+	/// Returns the delay for the given attempt count. If None, no delay is applied.
+	fn next_attempt_delay(retry_attempts: Self::AttemptCount) -> Option<Self::BlockNumber>;
+}
+
+pub struct DefaultRetryPolicy;
+impl RetryPolicy for DefaultRetryPolicy {
+	type BlockNumber = u32;
+	type AttemptCount = u32;
+
+	fn next_attempt_delay(_retry_attempts: Self::AttemptCount) -> Option<Self::BlockNumber> {
+		Some(10u32)
```

Comment:
I want to review these numbers... Defaulting to 10 blocks doesn't seem right.

Reply:
yeah the number can be updated after the system is implemented in a separate PR. 

As I mentioned to dan:
"for the bitcoin, the delay caps off 35 failed attempts, and is gonna be 2 hours delay for each retry after wards, which means it's gonna take 115 * 2 hours (~10 days) to abort for 150 validators, assuming each validator immidiately report failure and not time out, not sure if this is what we want"